src/solace_agent_mesh/agent/sac/app.py

```diff
@@ -353,9 +353,9 @@ class SamAgentAppConfig(SamConfigBase):
         description="Configuration for the agent's custom cleanup function.",
     )
     text_artifact_content_max_length: int = Field(
-        default=1000,
+        default=100000,
```

Comment:
Since we are adding one constant here (which is good), both the max and default should be constants in the constants file